apps/utils/locust_tests/lambda/server-regression/app.py

```diff
@@ -9,7 +9,11 @@
 import boto3
 from botocore.config import Config
 
-environment = os.environ.get("BFD_ENVIRONMENT", "test")
+# TODO: environment may be "prod-sbx" or "prod_sbx" depending on context (specifically, Glue Table
+# partition columns) so a better way of handling its string representation should be considered.
+# For now, "prod-sbx" is the only string representation expected to be encountered by this code
+# and other contexts
```

Comment:
Seems more like a general comment than a TODO. Maybe its worth just checking for the env and normalizing it if its prod-sbx or prod_sbx into one value here, so we don't even have to worry about this.

Reply:
I agree - I did want to talk about this. It's not a new comment, but I didn't want to remove it either or make any assumptions re: normalizing the value because there are many instances where `prod-sbx` would automatically get converted to `prod_sbx` by AWS/TF, so if it *is* important to distinguish the two, we'll need something more intelligent than a dash-to-underscore conversion here.

Reply:
Yeah if its too complex to include as part of this it's fine, but does seem like a good cleanup to do to avoid some future footgun.